FirebaseStorage/Sources/FIRStorage_Private.h

```diff
@@ -29,6 +31,27 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property(strong, nonatomic) NSString *storageBucket;
 
+/**
+ * Maximum time between retry attempts for uploads.
+ *
+ * This is used by GTMSessionFetcher and translated from the user provided `maxUploadRetryTime`.
+ */
+@property NSTimeInterval maxUploadRetryInterval;
+
+/**
+ * Maximum time between retry attempts for downloads.
+ *
+ * This is used by GTMSessionFetcher and translated from the user provided `maxDownloadRetryTime`.
+ */
+@property NSTimeInterval maxDownloadRetryInterval;
+
+/**
+ * Maximum time between retry attempts for any operation that is not an upload or download.
+ *
+ * This is used by GTMSessionFetcher and translated from the user provided `maxOperationRetryTime`.
+ */
+@property NSTimeInterval maxOperationRetryInterval;
```

Comment:
These properties should all be `(nonatomic, assign)`.

Reply:
Done